### PR TITLE
Added 3 tunisian restaurants

### PR DESCRIPTION
**Pull request by eddy-dardour**
Hi I am new to GitHub and also new to OpenSource, I tried to add three restaurants in the restaurant.json file. Comment if I made a mistake, I am not very experienced.

### CODE REVIEW

**Comment by arch0345** [comment]
Thanks and welcome to the NSI community! It looks like you added a file in the parent directory - is it possible for you to add these to the existing file in another PR (`data/brands/amenity/restaurant.json`)? To edit it, just click on the pencil icon. If you need any help feel free to respond

**Comment by eddy-dardour** [comment]
I don't really get what you mean. I actually just changed the json file.

**Comment by arch0345** [comment]
The file should be in the appropriate folder within the data folder (in this case `data/brands/amenity`) instead of in the root (where the `app` folder, `data` folder, and others are located).

**Comment by LaoshuBaby** [comment]
图片

Hey! you need to move it to /data/brands/amenity folder

If you don't know how to do, you can see this commit https://github.com/osmlab/name-suggestion-index/commit/6c968aa28a50cae5e6ee2dfc14fb4bc0aa9a112a .

**Comment by LaoshuBaby** [comment]
Besides, according to the text diff, you've add 3 brands:

+ Baguette & Baguette
+ Echemi
+ Plan B

If you don't add a wikidata in those entry, they will not be suggested by iD/RapiD automatically.

**Comment by arch0345** [comment]
Thanks for the quick fix @LaoshuBaby! This PR looks good, just note that these will not be generated as a preset in iD without a `brand:wikidata` tag

File: data/brands/amenity/restaurant.json [hunk 1]
```diff
@@ -193,6 +193,17 @@
         "official_name": "Bahama Breeze Island Grille"
       }
     },
+    {
+      "displayName": "Baguette & Baguette",
+      "id": "eatnpark-96af40",
+      "locationSet": {"include": ["tn"]},
+      "tags": {
+        "amenity": "fast_food",
```
Comment (arch0345) [inline data/brands/amenity/restaurant.json:201]:
```suggestion
        "amenity": "fast_food",
```
This should probably go in `fast_food.json` instead

File: data/brands/amenity/restaurant.json [hunk 2]
```diff
@@ -3068,6 +3089,16 @@
         "name": "PizzaExpress"
       }
     },
+    {
+      "displayName": "Plan B",
+      "locationSet": {"include": ["fr"]},
+      "tags": {
+        "amenity": "fast_food",
```
Comment (arch0345) [inline data/brands/amenity/restaurant.json:3096]: Same thing applies here

**Review by arch0345 — CHANGES_REQUESTED** [review]
The fast food brands should be moved from restaurant.json to fast_food.json

**Comment by eddy-dardour** [comment]
Alright I'll do the following changes

**Comment by LaoshuBaby** [comment]
> * Baguette & Baguette
> 
> * Echemi

Those 2 was added in #5733 

（Maybe because “Plan B” haven‘t got a wikidata item yet. You can try to create one if you like.）

Thank you for making your first contribution to NSI ♥

**Pull request closed without merging**